akka-docs-dev/rst/scala/code/docs/stream/FlowErrorDocSpec.scala

```diff
@@ -75,7 +77,7 @@ class FlowErrorDocSpec extends AkkaSpec {
         if (elem < 0) throw new IllegalArgumentException("negative not allowed")
         else acc + elem
       }
-      .withAttributes(OperationAttributes.supervisionStrategy(decider))
+      .withAttributes(ActorOperationAttributes.supervisionStrategy(decider))
```

Comment:
Can we abbreviate? ActorAttributes? Maybe we should rename OperationAttributes, too.